tool/tctl/common/resource_command.go

```diff
@@ -1042,6 +1043,11 @@ func (rc *ResourceCommand) Delete(ctx context.Context, client auth.ClientI) (err
 			return trace.Wrap(err)
 		}
 		fmt.Printf("cluster auth preference has been reset to defaults\n")
+	case types.KindClusterMaintenanceConfig:
+		if err := client.DeleteClusterMaintenanceConfig(ctx); err != nil {
```

Comment:
any consequences for deleting it? I see we are doing a reset for other configs instead.

Reply:
The config isnt created by default so resetting it to an empty version doesnt seem right, but I'm not sure what the correct behavior should be here

Reply:
As long as the caller is ok with not found. Seems fine:
https://github.com/gravitational/teleport/blob/64d3e8c505c11159ab795fc33d4138b6632fc5db/lib/auth/auth.go#L825-L833